xarray/test/test_plot.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import division
```

Comment:
Can you add this to the top of literally every single Python file in xarray?

That will keep this easier to reason about.